robotbase/src/main/java/com/team2357/frc2022/Constants.java

```diff
@@ -251,4 +251,12 @@
     public final class KICKER {
         public static final double SPEED = 0;
     }
+
```

Comment:
Maybe instead of having this as its own class, make it an inner class of the ```Constants.SHOOTER``` class.